src/current/v23.2/create-virtual-cluster.md

```diff
@@ -0,0 +1,110 @@
+---
+title: CREATE VIRTUAL CLUSTER
+summary: The CREATE VIRTUAL CLUSTER statement creates a new target virtual cluster for physical replication.
+toc: true
+docs_area: reference.sql
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+{% include enterprise-feature.md %}
+
+{% include_cached new-in.html version="v23.2" %} The `CREATE VIRTUAL CLUSTER` statement starts a [physical cluster replication job]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}). `CREATE VIRTUAL CLUSTER` only supports starting a replication stream as part of the [physical cluster replication]({% link {{ page.version.version }}/physical-cluster-replication-overview.md %}) workflow.
```

Comment:
This isn't strictly true. The first variant of the statement in the diagram doesn't start a physical replication job.  If we want to restrict the document to that, then I think we should exclude the variant that doesn't include `FROM REPLICATION OF` from the diagram if possible. If we are going to document both, I might say something like:

```
The CREATE VIRTUAL CLUSTER statement creates a new virtual cluster.  It is only currently only supported for use in [physical cluster replication job]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}).
```

Reply:
I have changed the paragraph to your suggestion... I'm not sure if to remove just the `CREATE VIRTUAL CLUSTER` from the synopsis diagram or not. I did not create an example for that on purpose. If users could cause themselves problems, if they do run `CREATE VIRTUAL CLUSTER blah` then I can remove... if we think it won't cause much harm then I will leave as is

Reply:
Checked in with @stevendanna re this on Slack, we're going to leave the diagram as-is for now.